README.md

```diff
@@ -72,7 +72,7 @@ See [Notebooks](#Notebooks) section for further illustrations.
 
 ## Notebooks
 
-_Once the library is successfully installed, please RESTART your Jupyter kernels or Google Colab runtimes for the changes to take effect._
+_After succesful installtion of library, RESTART Jupyter kernels or Google Colab runtimes for the changes to take effect._
```

Comment:
> After succesful installtion of library
After successful installation of library

Reply:
> After successful installation of library
After successful installation of the library -- my bad I didn't notice @dA505819

Reply:
Done